src/scripts/organizeInternInfo.js

```diff
@@ -0,0 +1,22 @@
+export default function organizeInternInfo(intern) {
+  const col = document.createElement("td"); //Create column for intern
+  const internInfo = document.createElement("div"); //Column info div
+  internInfo.className = "intern-pill-name-location";
+
+  const pill = document.createElement("div"); //Department pill div
+  pill.className = "pill";
+  pill.innerHTML = "<b>" + intern.department + "</b>";
```

Comment:
this is a nit but you can use [template strings ](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals)for this 
```suggestion
  pill.innerHTML = `<b>${intern.department}</b>`;
```